### ISSUE TITLE
☔Performance Variability

### ISSUE THREAD

**Issue by patrickhulce** [GoogleChrome/lighthouse#10657]
## tl;dr - follow the advice in the [variance docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md)

You're probably here because you filed an issue wondering why metrics or performance results were different between two different runs. We are deeply appreciative of your effort to improve Lighthouse by letting us know!

Performance variability of webpages is a very challenging topic, especially if Lighthouse is one of your first experiences with performance measurements. We've documented all of the most common sources of performance variability in our [variance docs](https://github.com/GoogleChrome/lighthouse/blob/master/docs/variability.md) as well as steps you can take to limit its impact.

Lighthouse has a few internal mechanisms that attempt to limit the impact of variability such as [simulated throttling](https://github.com/GoogleChrome/lighthouse/blob/master/docs/throttling.md), `CPU / Memory Power` estimation in the runtime section of the report, and companion projects like [lighthouse-ci](https://github.com/GoogleChrome/lighthouse-ci/) that can automatically run Lighthouse many times. Ultimately though, there is only so much that can be done from within Lighthouse itself and the eventual results rely on the stability of the environment in which Lighthouse is run.
